frontend-new/components/LoginPage.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Typography, Link } from '@mui/material'
+import { FC } from 'react'
+
+import { LinkIconButton } from 'app/components'
+import { Google, Microsoft } from 'app/icons'
+import { media } from 'app/styles/theme'
+
+export const LoginPage: FC = () => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+      }}
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          flexDirection: 'column',
+          backgroundColor: '#F4F3FF',
+          gap: 3,
+          color: '#343AAA',
+
+          width: 276,
+          height: 276,
+          mt: 4,
+          px: 5.5,
+
+          [media.TABLET]: {
+            mt: 8,
+            width: 379,
+            height: 352,
+            px: 9,
+          },
+
+          [media.DESKTOP]: {
+            mt: 8,
+            width: 432,
+            height: 436,
+            px: 10.5,
+          },
+        }}
+      >
+        <Typography variant='h1'>Sign In</Typography>
+        <LinkIconButton
+          icon={<Google />}
+          href='/_/auth/login/google?redirect_to=https%3A%2F%2Ftrot.to'
```

Comment:
Would these login URLs be better managed in a config lookup?

Reply:
Yeap, good catch.